### PR TITLE
fix path in CI for release & python version in all dockers & duplicated badges

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

- fix path typo in nightly CI so it is really released
- fixed python version in all docker images
- probably fixed PT version in dockers too
- added final check for versions
- duplicate badges...

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
@zcain117 mind checking it... I have also added a check ad the docker end that versions are correct

File: dockers/base-cuda/Dockerfile [hunk 1]
```diff
@@ -74,26 +90,28 @@ ENV CONDA_DEFAULT_ENV=${CONDA_ENV}
...
 COPY ./requirements/test.txt requirements-tests.txt
-COPY ./requirements/examples.txt requirements-examples.txt
 
 RUN \
     # Disable cache
     pip config set global.cache-dir false && \
     #echo ". ${WORKDIR}/miniconda/etc/profile.d/conda.sh" >> ~/.bashrc && \
     #echo "conda activate ${CONDA_ENV}" >> ~/.bashrc && \
     #source ~/.bashrc && \
+    # install NVIDIA AMP
```
Comment (Borda) [inline dockers/base-cuda/Dockerfile:100]: @ydcjeff :]

Comment (ydcjeff) [reply dockers/base-cuda/Dockerfile:100]: Ouch it took 20mins like before

What abt skipping apex in py3.8 since we don't use py3.8 in Drone?

Comment (Borda) [reply dockers/base-cuda/Dockerfile:100]: we want to give Docker users full experience, that is the point to have installed all long-running tasks, also this mostly run in a nightly fashion/CI and at this moment Drone takes quite long with some queue...

Comment (ydcjeff) [reply dockers/base-cuda/Dockerfile:100]: Okay 👌

**Review by justusschock — APPROVED** [review]

**Review by ydcjeff — APPROVED** [review]
LGTM

**Review by SkafteNicki — APPROVED** [review]
LGTM

**Pull request merged**